middleware.js

```diff
@@ -12,13 +14,33 @@ module.exports.isLoggedIn = (req, res, next) => {
 };
 
 module.exports.fetchRandomGameData = async (req, res, next) => {
-  const randGameId = Math.floor(Math.random() * 1000 + 1);
-  const results = await axios.get(
-    `https://api.rawg.io/api/games/${randGameId}?key=${process.env.RAWG_KEY}`
-  );
-  req.gameData = results.data;
+  let response = null;
+  let retries = 1;
+  let success = false;
+  const maxRetries = 5;
 
-  next();
+  while (retries <= maxRetries && !success) {
+    try {
+      const randGameId = Math.floor(Math.random() * 500000 + 1);
+      response = await axios.get(
+        `https://api.rawg.io/api/games/${randGameId}?key=${process.env.RAWG_KEY}`
+      );
+      success = true;
+      break;
+    } catch (error) {
+      console.log(error.response.status);
```

Comment:
For the future let's implement showing the error in console only if in development

Reply:
added a .env variable together with conditionals for some console.logs like this one